public/robots.txt

```diff
@@ -1 +1,4 @@
 # See https://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
+
+user-agent: *
+disallow: /*.m3u8
```

Comment:
After looking over the documentation, I don't think this will work. 

> Note also that globbing and regular expression are not supported in either the User-agent or Disallow lines. The '\*' in the User-agent field is a special value meaning "any robot". Specifically, you cannot have lines like "User-agent: *bot*", "Disallow: /tmp/*" or "Disallow: *.gif".

With that, I think we would need to disallow each possible m3u8 endpoint so something like:
```suggestion
disallow: /index.m3u8
disallow: /auto.m3u8
disallow: /high.m3u8
disallow: /medium.m3u8
disallow: /low.m3u8
```

Reply:
I was looking at Google's docs and it allows wildcards: https://developers.google.com/search/docs/crawling-indexing/robots/robots_txt

From reading that page it looks like specifying each route probably won't work for us because the disallow isn't relative so we would need to know the masterfile ids which leads back to the wildcard.